content/en/docs/contribute/style/style-guide.md

```diff
@@ -49,17 +49,18 @@ When you refer specifically to interacting with an API object, use [UpperCamelCa
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-Refer to API objects without saying "object," unless omitting "object"
-leads to an awkward construction.
+There is a subtle distinction between formal API objects and general concepts. Use "object" where appropriate to clarify the meaning of terms. Calling out a "Pod object" may be appropriate when moving from general concepts to specific API objects. For example, "Pods may have multiple volume mounts. The Pod object includes a Volumes field."
+
+Do not use "object" repetitively, such as where it's clear a term is an API object. 
+
+Note that capitalization is a weak way to signal information to the reader. This information may be lost entirely due to assistive technology, differences in visual abilities, or translation. 
 
 {{< table caption = "Do and Don't - API objects" >}}
 Do | Don't
 :--| :-----
-The pod has two containers. | The Pod has two containers.
-The HorizontalPodAutoscaler is responsible for ... | The HorizontalPodAutoscaler object is responsible for ...
 A PodList is a list of pods. | A Pod List is a list of pods.
-The two ContainerPorts ... | The two ContainerPort objects ...
-The two ContainerStateTerminated objects ... | The two ContainerStateTerminateds ...
+HostPath is a type of Volume. | Hostpath is a type of volume.
+The Volume object is tied to a Pod. | The volume is tied to a pod.
 {{< /table >}}
```

Comment:
I'm open to feedback on the revised do/don't table

Reply:
The example on line 63 is a little confusing to me, a non-native English speaker. I'm okay to this sentence:

> The Volume object is referenced by a Pod.

Because in this context, saying "The Volume is referenced by a Pod" instead is not acceptable, because we are obviously talking about the Volume API object.

As a second example, the following may be NOT okay:

> That Volume object is mounted when the Pod is created.

The "object" should be deleted here because we are talking about the storage volume represented by a Volume.

You example is confusing to me because the meaning of "tied to" is vague.

Reply:
It's valuable information that the proposed example is unclear to non-native English speakers. I am revising the examples based on Tim's suggestions, and I think that will solve this problem. Thanks!